From: Mangal Kushwah <mangalk2324@gmail.com>
Date: Thu, 8 Aug 2024 21:30:19 +0530
Subject: demux: AVI insert packetizer for XVID MPEG-4 video

Older DivX-encoded videos commonly use an method called packed
bitstream which puts several video frames into a single AVI chunk.
Since Packed bitstream isn't standard MPEG-4 it causes playback issues
with some hw decoders.

(cherry picked from commit 980e4d7449e9e06bef525d9049dff5b9b5ecd673)
---
 modules/demux/avi/avi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/demux/avi/avi.c b/modules/demux/avi/avi.c
index 7059ca9..bcc3511 100644
--- a/modules/demux/avi/avi.c
+++ b/modules/demux/avi/avi.c
@@ -719,6 +719,7 @@ static int Open( vlc_object_t * p_this )
                     {
                         tk->fmt.i_codec           =
                         tk->fmt.i_original_fourcc = VLC_FOURCC( 'X', 'V', 'I', 'D' );
+                        tk->fmt.b_packetized = false;
                     }
 
                     /* Shitty files storing chroma in biCompression */
